tox.ini

```diff
@@ -56,7 +56,7 @@ commands =
     # can not use a different configuration directory vs source directory on RTD
     # currently -- https://github.com/rtfd/readthedocs.org/issues/1543.
     # That is why we have a "-c docs/html" in the next line.
-    sphinx-build -W -d {envtmpdir}/doctrees/man -b man docs/man docs/build/man -c docs/html
+    sphinx-build -W -d {envtmpdir}/doctrees/man -b man docs/man docs/build/man -c docs/html -D extensions=sphinx.ext.extlinks,pip_sphinxext,sphinx.ext.intersphinx
```

Comment:
We should set `sphinx_tabs_nowarn = True` in conf.py, instead of modifying this.

Reply:
I have removed the changes to tox.ini and updated conf.py to add `sphinx_tabs_nowarn = True`